src/molecules/PriceDiscountHeader/__tests__/price_discount_header.spec.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import PriceDiscountHeader from '../index';
+
+describe('PriceDiscountHeader', () => {
+
+  it('is a <PriceDiscountHeader>', () => {
+    const wrapper = shallow(<PriceDiscountHeader />);
+
+    expect(wrapper.length).to.equal(1);
+    expect(wrapper.find('.label')).to.have.length(0);
+    expect(wrapper.find('.prev-amount')).to.have.length(0);
+  });
+
+  it('shows label if prop.label is set', () => {
+    const wrapper = shallow(<PriceDiscountHeader amount={100} label='stuff' />);
+
```

Comment:
I suggest that we don't test for `.label` and `.prev-amount` to be length 0 in the previous test and to test them in their respective `it` blocks instead. You could omit them for the initial render, test they are 0, then set props to include them & test for length 1. example: https://github.com/policygenius/athenaeum/blob/master/src/atoms/Button/__tests__/button.spec.js#L35